.github/workflows/pr.yml

```diff
@@ -132,9 +132,13 @@ jobs:
           pre-commit run --all-files flake8
           pre-commit run --all-files prettier
           pre-commit run --all-files check-yaml
+      - name: Setup
+        uses: ./.github/actions/rust
       - name: Cargo fmt check
         shell: bash
         run: cargo fmt -- --check
+      - name: Clippy
+        run: cargo clippy --all-targets --all-features --keep-going -- -D warnings
```

Comment:
maybe we should modify the steps in this workflow to always run, even if one errors?

Reply:
Good point. Updated so that it continues-on-error for non-setup steps so that if setup fails, the job fails.